www/performance_optimization.php

```diff
@@ -123,9 +123,10 @@
             }
             ?>
 
+            <h3 class="hed_sub">Glossary</h3>
             <?php
-                echo '<h3 class="hed_sub">Optimization Details</h3>';
-                dumpOptimizationGlossary();
+                require_once __DIR__ . '/resources/view.php';
+                echo view('snippets.glossary', []);
```

Comment:
Do we want to namespace `view` so we can use `use` with it?

Reply:
hmm, maybe. The thing is it's like this in Laravel so I thought it'd be easier to replace.

Also `/resources/view.php` is now autoloaded in a newer PR so `view()` would be just a globally accessible function like it is in Laravel: https://laravel.com/docs/9.x/blade#introduction